download and run gradle build fail

step 1. download the package and extract 

step 2. run gradle build 
[...java/cucumber-java-skeleton]$ gradle build

> Task :test

skeleton.RunCukesTest > initializationError FAILED
    java.lang.ExceptionInInitializerError
        Caused by: cucumber.runtime.CucumberException

1 test completed, 1 failed



i am new for cucumber and java, how should i make the skeleton work

There is a subtle difference between the maven and gradle outputs. 

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> One or more feature files failed to parse. See error output above
```

It tries to parse a feature file. Running with `--debug --stacktrace` shows there is another plugin involved that tries to run cucumber directly rather then through JUnit.

```
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Caused by: org.gradle.api.GradleException: One or more feature files failed to parse. See error output above
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at com.commercehub.gradle.cucumber.CucumberRunner.run(CucumberRunner.groovy:98)
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at com.commercehub.gradle.cucumber.CucumberRunner$run.call(Unknown Source)
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at com.commercehub.gradle.cucumber.CucumberTask.runTests(CucumberTask.groovy:40)
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:75)
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.doExecute(DefaultTaskClassInfoStore.java:136)
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.execute(DefaultTaskClassInfoStore.java:129)
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.execute(DefaultTaskClassInfoStore.java:118)
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:623)
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:606)
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:80)
23:01:43.390 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:61)
```

I reckon removing `com.commercehub.cucumber-jvm` and related config from gradle would solve this problem.

--- a/README.md
+++ b/README.md
@@ -45,7 +45,7 @@ If you remove it (and the `cucumber-junit` jar dependency), it will run just the
 
 Open a command window and run:
 
-    gradlew test
+    gradlew test --info
 
 This runs Cucumber features using Cucumber's JUnit runner. The `@RunWith(Cucumber.class)` annotation on the `RunCukesTest`
 class tells JUnit to kick off Cucumber.

--- a/build.gradle
+++ b/build.gradle
@@ -1,18 +1,4 @@
-buildscript {
-    repositories {
-        maven {
-            url "https://plugins.gradle.org/m2/"
-        }
-    }
-    dependencies {
-        classpath "com.commercehub:gradle-cucumber-jvm-plugin:0.13"
-    }
-}
-
-plugins {
-    id 'java'
-    id 'com.commercehub.cucumber-jvm' version '0.13'
-}
+apply plugin: 'java'
 
 project.ext {
     cucumberVersion = '2.3.1'
@@ -29,16 +15,4 @@ dependencies {
 
 repositories {
     mavenCentral()
-}
-
-addCucumberSuite 'test'
-
-cucumber {
-    maxParallelForks = 3
-}
-
-test {
-    stepDefinitionRoots = ['skeleton']
-    featureRoots = ['skeleton']
-    plugins = ['io.cucumber.pro.JsonReporter:full-gradle']
 }

